features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/data/AuthManager.js

```diff
@@ -190,16 +198,17 @@ class AuthManager {
     logout() {
         let authHeader = "Bearer " + AuthManager.getUser().getPartialToken();
         //TODO Will have to change the logout end point url to contain the app context(i.e. publisher/store, etc.)
-        let url = this.host + "/login/logout" + this.contextPath;
+        let url = Utils.getAppLogoutURL()
         let headers = {
             'Accept': 'application/json',
             'Content-Type': 'application/json',
             'Authorization': authHeader
         };
         const promisedLogout = axios.post(url, null, {headers: headers});
         return promisedLogout.then(response => {
-            Utils.delete_cookie("WSO2_AM_TOKEN_1");
-            localStorage.removeItem("wso2_user");
+            Utils.delete_cookie("WSO2_AM_TOKEN_1", Utils.CONST.CONTEXT_PATH);
```

Comment:
Shall we use a constant for `WSO2_AM_TOKEN_1` cookie name ?

Reply:
Fix with the commit https://github.com/RenukaFernando/carbon-apimgt/commit/685f9e552b3bb256db42d4840496ce259ccd3237